.github/workflows/ci.yaml

```
@@ -30,6 +30,6 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 16
+          node-version: 18
```

In the current version, there is `[14, 16]` and `16`. I would have used versions `[18, 20]` and **`20`**.

```suggestion
          node-version: 20
```